src/subclassers.ts

```diff
@@ -61,3 +79,11 @@ export function WithResponse<TWhereToLook extends string>(
     },
   };
 }
```

Comment:
- [ ] why's this name weird?

Reply:
i want to name it WithResponse but this name is already taken
so i decide to make it WithSafeResponse since this function return safe response

Reply:
ok we'll deal with it on a grand naming cleanup task later ;)